modules/performing-an-image-signature-verification.adoc

```diff
@@ -0,0 +1,123 @@
++
+For more information, go to the GitHub repository at `google/re2`, open the `Wiki` section, and then select the `Syntax` page.
+
+.. Enter the certificate identity that Cosign specifies. You must use regular expressions in RE2 syntax for matching.
++
+For more information, go to the GitHub repository at `google/re2`, open the `Wiki` section, and then select the `Syntax` page.
+.. Enter the trusted certificate root which is encoded in PEM format to verify the certificates. If you do not specify the certificate root, the public Fulcio roots are used automatically for verification. 
++
+For more information, see link:https://docs.sigstore.dev/certificate_authority/overview/[Fulcio] (Sigstore documentation).
+
+.. Enter the trusted signer intermediate certificate authority to verify the certificates. If you do not specify the certificate authority, the certificate chain is used automatically for verification.
+.. Optional: Select the *Enable certificate transparency log validation* checkbox to validate the proof of inclusion into the certificate transparency log.
++
+Enter the public key that you want to use to validate the proof of inclusion into the certificate transparency log.
+If you do not specify the public key, the key of the public Sigstore instance is used automatically for validation.
+
+. Optional: To configure the transparency log, complete the following steps:
++
+[NOTE]
+====
+When you create a signature integration, you can enable the validation of transparency logs in the following situations:
+
+** When the signatures contain short-lived certificates that Fulcio issues.
+** When you want to use keyless verification of the signatures.
+** To verify the signatures, when you use a public key.
+====
+.. Select the *Enable transparency log validation* checkbox to validate the inclusion of the signature in a transparency log.
++
+Enter the URL where the Rekor transparency log is available. If you do not specify the URL, the public Rekor instance of Sigstore is used automatically for validation.
++
+[NOTE]
+====
+The Rekor URL is required for online confirmation of the inclusion into the transparency log.
+====
+.. Optional: Select the *Validate in offline mode* checkbox to force the offline validation of the signature proof of inclusion into the transparency log.
++
+[NOTE]
+====
+You can force the offline validation of the signature proof of inclusion into the transparency log only if you have enabled the validation of the transparency log.
+====
++
+Enter the public key to validate the signature proof of inclusion into the Rekor transparency log.
+If you do not specify the public key, the key of the public Sigstore instance is used automatically for validation.
+
+. Click *Save*.
+
+.Verification
+
+* Verify that you can perform an image signature verification:
+** To scan an image signature by using the `roxctl` CLI, run the following command:
```

Comment:
This looks like the actual procedure for performing an image signature verification. Is the rest of the content in this module really prereq for this?